spec/prism_scanner_spec.rb

```diff
@@ -346,9 +346,9 @@ def value
         expect(occurrence.path).to eq(
           'spec/fixtures/used_keys/app/controllers/a.rb'
         )
-        expect(occurrence.line_num).to eq(7)
+        expect(occurrence.line_num).to eq(4)
         expect(occurrence.line).to eq(
-          "# i18n-tasks-use t('translation.from.comment3')"
+          "I18n.t(\"scoped.translation.\#{variable}\")"
```

Comment:
This makes it so it's not longer as obvious in the report that the calls are coming from a magic comment.
Not sure how big of a deal that is.